### PR TITLE
Kickerz/hide members for not logged in

### PR DESCRIPTION
**Pull request by niconomaa**
Hide form containing table of team members on team show page from users who are not logged in.
@reviewers: Use https://github.com/hpi-swt2/sport-portal/pull/508/files?w=1 to see the actual diff without whitespace changes.

### CODE REVIEW

File: spec/features/team/show_team_spec.rb [hunk 1]
```diff
@@ -61,4 +61,19 @@
       expect(page).to have_link(I18n.t("helpers.links.email"), href: mailto_link_for_user)
     end
   end
+
+  describe 'display team members only for logged in users' do
+
+    it 'should not display team member table for users who are not logged in ' do
+      sign_out @user
```
Comment (lheilmann) [inline spec/features/team/show_team_spec.rb]: @tomkellog I would assume `@user` is not signed in, hence there is no need to sign out the `@user`. Can this line be deleted?

Comment (lheilmann) [reply spec/features/team/show_team_spec.rb]: Or you add the following block and adjust both test cases:
```
before(:each) do
    sign_in @user
end
```

File: spec/features/team/show_team_spec.rb [hunk 2]
```diff
@@ -61,4 +61,19 @@
...
+      visit team_path @team
+      expect(page).to_not have_css("table.table-striped")
```
Comment (lheilmann) [inline spec/features/team/show_team_spec.rb]: @tomkellog May not be the best approach to check on a table with no specific id. Imagine there are going to be multiple tables with the css-class `.table-striped` on this page... What about giving the table a meaningful id?

**Review by lheilmann — COMMENTED** [review]
Some minor thoughts. Rest looks good to me :beach_umbrella: Tested in browser, too!

**Review by bakoe — COMMENTED** [review]
LGTM code-wise, haven't tested in Browser though – I agree with the changes requested by @lheilmann.

**Review by bakoe — APPROVED** [review]

File: spec/features/team/show_team_spec.rb [hunk 3]
```diff
@@ -61,4 +61,18 @@
...
+
+    it 'should not display team member table for users who are not logged in ' do
+      visit team_path @team
+      expect(page).to_not have_css("table#members")
+    end
+
+    it 'should display team members table for users who are logged in' do
+      sign_in @user
+      visit team_path @team
+      expect(page).to have_css("table.table-striped")
```
Comment (bakoe) [inline spec/features/team/show_team_spec.rb]: Please use the newly introduced CSS ID here as well.

**Review by bakoe — CHANGES_REQUESTED** [review]

**Review by lheilmann — COMMENTED** [review]
You will have my GO, when you applied the changes @bakoe has requested.

File: spec/features/team/show_team_spec.rb (hunk 3 above)
Comment (niconomaa) [reply spec/features/team/show_team_spec.rb]: ooooops

Comment (niconomaa) [reply spec/features/team/show_team_spec.rb]: done

**Review by lheilmann — APPROVED** [review]
1A!

**Review by bakoe — APPROVED** [review]

**Pull request merged**